### PR TITLE
Add a dockerfile for easy development

### PR DESCRIPTION
**Pull request by akofink**
Signed-off-by: Andrew Kofink <akofink@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,16 @@ After checking out the repo, run `bin/setup` to install dependencies. Then, run
...
 To install this gem onto your local machine, run `bundle exec rake install`. To release a new version, update the version number in `version.rb`, and then run `bundle exec rake release`, which will create a git tag for the version, push git commits and tags, and push the `.gem` file to [rubygems.org](https://rubygems.org).
 
+### With Docker
+
+A Dockerfile is provided to allow a containerized development environment:
+
+```
+docker build . -t openscap_parser # build the container image
+docker run -itv $PWD:/app:z openscap_parser rake # run tests
+docker run -itv $PWD:/app:z openscap_parser rails c # console
```
Comment (xprazak2) [inline README.md]: `rails c` is not available

**Comment by xprazak2** [comment]
Nice work, I got git error during build but everything worked smoothly. Any chance it could be done without the error?

build-parser

**Comment by akofink** [comment]
Updated! The error with git is fixed by adding the entire repo to the image. The console command is updated for use with a gem (lol sorry for that oversight)

**Comment by xprazak2** [comment]
Thanks!

**Pull request merged**